### PR TITLE
Maya: load image plane set colorspace

### PR DESCRIPTION
**Pull request by BigRoy**
## Changelog Description

Set colorspace on loading image plane.

## Additional info

n/a

## Testing notes:

1. Load image plane should work
2. If loaded content has colorspace data (from publish) it should set the colorspace automatically.

### CODE REVIEW

File: client/ayon_core/hosts/maya/plugins/load/load_image_plane.py [hunk 1]
```diff
@@ -145,6 +145,18 @@ def load(self, context, name, namespace, data, options=None):
                 fileName=context["representation"]["data"]["path"],
```
Comment (MustafaJafar) [inline client/ayon_core/hosts/maya/plugins/load/load_image_plane.py:145]: This PR didn't work for me until I made that change on my side. 
```suggestion
                fileName=context["representation"]["attrib"]["path"],
```

Comment (BigRoy) [reply client/ayon_core/hosts/maya/plugins/load/load_image_plane.py:145]: Interesting - that's actually wrong too! :)

```suggestion
                fileName=self.filepath_from_context(context),
```

Comment (BigRoy) [reply client/ayon_core/hosts/maya/plugins/load/load_image_plane.py]: @MustafaJafar could you retry now with https://github.com/ynput/ayon-core/pull/246/commits/e78b8043ab9bf2e4e44d1e545e0ed64b9745c3c4

**Review by MustafaJafar — APPROVED** [review]
It works on my side. 
> I don't have render with colorspace data. Thus, I didn't if setting colorspace works.

image

**Review by tokejepsen — APPROVED** [review]
Tested successfully in Maya 2023.

**Pull request merged**